src/main/webapp/app/view/project/admin/users/project-user-invite-delete-dialog.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="row justify-content-center">
+    <div class="col-10">
+        <form *ngIf="invitation" name="deleteForm" (ngSubmit)="confirmDelete(invitation?.id!)">
```

Comment:
`invitation` must be present in this case (use `!`). or add this to the parent element:
```html
    <div class="col-10" *ngIf="invitation">
```